### DISCUSSION TITLE
Request translations from a different language than current

### DISCUSSION THREAD

**Discussion by vdlbk** [tolgee/tolgee-platform#1634] [Q&A] [answered]
Hello, 

I was wondering if it was possible to request a translation from a different language rather than the current language.
Let's say, the current language is `English` and I have a version in `Spanish`.
If I detect that my user's browser is in `Spanish`, I'd like to open a popup in Spanish to ask the user if he/she wants to switch to Spanish instead of English.

I guess it would be like: `$t('title', {lang:'es'})`.
Is it possible/supported?

I probably can solve this by having more keys, but as I'm trying to reduce the number of keys, I'm looking for alternative

What I think I should do : `$t('title.es')`
What I would like: `$t('title', {lang:'es'})`

Hope this question was not answered yet, was not able to find it.
Thanks!

**Comment by stepan662** [comment, marked as answer]
Hey, this is unfortunately not supported now. However it's an interesting usecase. I've created an issue for that and we'll think about this for the future ...
https://github.com/tolgee/tolgee-js/issues/3181
